diagnostic.md

```diff
@@ -40,15 +42,15 @@ methods.
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+ It grants JS access to the html and lets JS do whatever it wants to do within its capability. Basically, like an interface where the two interacts!
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.